src/handlers/fix-request-body.ts

```diff
@@ -2,17 +2,25 @@ import type * as http from 'http';
 import type { Request } from '../types';
 import * as querystring from 'querystring';
 
+type BodyParserRequest = {
+  body?: Record<string, any>;
+};
+
 /**
  * Fix proxied body if bodyParser is involved.
+ *
+ * {@link https://www.npmjs.com/package/body-parser}
+ * {@link https://github.com/chimurai/http-proxy-middleware/pull/492}
  */
-export function fixRequestBody(proxyReq: http.ClientRequest, req: http.IncomingMessage): void {
-  const requestBody = (req as Request).body;
+export function fixRequestBody(proxyReq: http.ClientRequest, req: Request): void {
+  const requestBody = (req as BodyParserRequest).body;
```

Comment:
here's an express runtime workaround, with an express type cast, all localized to the module of interest